data/gegno/gegno intro missions.txt

```diff
@@ -439,6 +442,311 @@ mission "Giaru Gegno: Revisit"
+			`	The Scin then reaches their palm out and makes a mark on one half of the display, then turns it to you.`
+			choice
+				`	(Place your hand on the device.)`
+				`	(Do nothing.)`
+					goto "images"
+			`	Placing your own palm on the device, you feel it vibrate a little. A detailed wireframe of your hand appears on the other half of the device. The Scin moves the device aside, then extends one arm out showing the top of their hand.`
+			choice
+				`	(Extend your arm as well.)`
+				`	(Do nothing.)`
+					goto "images"
+			action
+				set "scin: gestures"
+			`	Reaching with your right hand, the Scin leans forward and carefully examines it, then makes another note on their device. After writing down a few more things, they turn to you gesture strangely, crossing their arms together in a plus-shaped fashion towards their left. The other heavier-armored Scin move closer, escort you back to the elevator and back to your ship, but along the way are a few Scin you haven't seen yet. Their armor is much different than that of the others, characterized by darker shades, sharper edges, and several gadgets, but what is most notable is their possession of sickle-esque blade weapons.`
+			`	Once you return to your ship, the Scin return to theirs, but remain on the planet. Although just as strange, this encounter went far better than the one with the Vi.`
+				decline
+
+			label "images"
+			`	Turning the device back around, the Scin types some sort of note on it then clears the display. They then begin to draw some sort of picture with one of their fingers. The finished image appears to be a portrait of themselves. "Dueyu Scin," they speak in a low tone. Clearing the image, they draw something that looks like you, albeit a bit out of proportion, then await some sort of answer.`
+			choice
+				`	"Dueyu human..?"`
+				`	"I am human."`
+				`	(Say nothing.)`
+					goto "silence"
+			`	The Scin looks at you in what you guess to be fascination. They grab the device and draw yet another picture. After setting it down, you recognize the familiar image of a Gegno Vi Warrior.`
+			choice
+				`	"Gegno Vi?"`
+				`	"Dueyu Vi?"`
+			action
+				set "scin: imagery"
+			`	Looking at you a bit confused, they draw another portrait that looks like the shorter leather-wearing Gegno from their homeworld, and move that image together with the images of the Vi and the Scin. "Gegno," they say, and begin to think to themselves. A brief moment passes, and they begin drawing on the device again. After turning it back to you, you see both a green-colored patch. and a brown-colored patch. "Vie," the Scin says, pointing to the brown. "Sinn," they say, now pointing to the green. Afterwards, they point to the group of the three Gegno portraits. "Gegno."`
```

Comment:
```suggestion
			`	Looking at you a bit confused, they draw another portrait that looks like the shorter leather-wearing Gegno from their homeworld, and move that image together with the images of the Vi and the Scin. "Gegno," they say, and begin to think to themselves. A brief moment passes, and they begin drawing on the device again. After turning it back to you, you see both a green-colored patch and a brown-colored patch. "Vi," the Scin says, pointing to the brown. "Scin," they say, now pointing to the green. Afterwards, they point to the group of the three Gegno portraits. "Gegno."`
```
Are these misspellings intentional? Phonetic, maybe?

Reply:
Yes, they're intentional. Similar to how different languages are spelled/said differently between countries (Ingles, etc..).